src/calculateSimilarity.ts

```diff
@@ -0,0 +1,39 @@
+import { disassembleHangul } from './disassemble';
+
+export function calculateSimilarity(a: string, b: string): number {
+  const disassembleAndClean = (text: string) => disassembleHangul(text).replace(/\s/g, '');
+
+  const disassembledA = disassembleAndClean(a);
+  const disassembledB = disassembleAndClean(b);
```

Comment:
XAndY 와 같은 함수명보다는 아래와 같이 사용하는 게 어떨까요?
https://careerly.co.kr/comments/103186

```suggestion
  const disassembledA = disassembleHangul(a).replace(/\s/g, '');
  const disassembledB = disassembleHangul(b).replace(/\s/g, '');
```